packages/odyssey-react/src/components/Select/Select/index.tsx

```diff
@@ -14,7 +14,7 @@ import React, { useCallback } from "react";
 import type { ChangeEvent, ReactElement, ComponentPropsWithRef } from "react";
 import { SelectOption } from "../SelectOption";
 import { SelectOptionGroup } from "../SelectOptionGroup";
-import { useChoices } from "./useChoices";
+import { ChoicesHTMLSelectElement, useChoices } from "./useChoices";
```

Comment:
minor, but consider importing the type separatly via the explicit type import:
```
import { useChoices } from './useChoices';
import type { ChoicesHTMLSelectElement } from './useChoices';
```
while it does require more typing its also more explicit about value level and type level concerns and guarantees that the references will be removed at compilation time which is what we want for types

Reply:
Agree the distinction is important, got used to the other repo where the import type / export type syntax can actually cause a failure 👎